src/main/java/com/webs/user/JwtFilter.java

```diff
@@ -0,0 +1,86 @@
+package com.webs.user;
+
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JWSVerifier;
+import com.nimbusds.jose.crypto.MACVerifier;
+import com.nimbusds.jwt.SignedJWT;
+import org.slf4j.Logger;
```

Comment:
Let's separate groups of imports pleases :D And minor, not sure if we want to implement a not-too-extensive level of checkstyle for handling import checks, additional checks... (http://checkstyle.sourceforge.net/config_imports.html).